velox/common/memory/ByteStream.h

```diff
@@ -362,20 +243,25 @@ class ByteStream {
   }
 
   StreamArena* arena_;
-  // Indicates that position in ranges_ is in bits, not bytes.
+
+  /// Indicates that position in ranges_ is in bits, not bytes.
```

Comment:
IIRC: we just use // for private section and only /// for public section. Thanks!

Reply:
Cool I'll change. Then we have a ton of code to refactor. I'm seeing many classes having /// for private sections.